src/styles/ant-vars.less

```diff
@@ -0,0 +1,222 @@
+/* Overrides styling for the default Ant Design theme found here:
+https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less */
+
+@dark: #141219;
+@dark-two: #1e1b25;
+@dark-three: #25222e;
+@dark-four: #2f2a3c;
+@white-six: #e7e7e7;
+@baby-purple: #b59ff6;
+@dark-purple: #3b3649;
+@allen-purple: #8d87aa;
+@greyish-brown: #4a4a4a;
+@highlight-green: #b2d030;
+@blue: #0094FF;
+
+// -------- Colors -----------
+@primary-color : @baby-purple;
```

Comment:
I like that this .less file is included in this template, but I wonder if it would be better start it off from a blank slate? I think a lot of the styling in here is specific to simularium-website. What I'm envisioning is a copy of https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less that is entirely commented out

Reply:
that's a good thought. The reason I put this here is because it seems like the simularium branding might become the Animated Cell branding, so we'd end up copying over a lot of color settings anyway? I dunno which is more likely

Reply:
I'm not sure how much we will even be using this project repo, but I imagined this file as sort of the AICS web front end style guide, containing our stock color palette etc.  It would be a starting point for further changes in specific projects

Reply:
Ok, I'm ok with this too :)